lib/tasks/csv_load.rake

```diff
@@ -2,6 +2,7 @@ require "csv"
 
 namespace :csv_load do
   task customers: [:environment] do
+    print "."
```

Comment:
When you run csv_load:all you'll see this in action!

Reply:
How did you figure this out?